_data/rules.yml

```diff
@@ -435,3 +435,11 @@ services:
           description: Interface is getting saturated for at least 1 min. (> 0.80GiB/s)
           query: 'irate(junos_interface_transmit_bytes[1m]) * 8 > 1e+9 * 0.80'
           severity: warning
+
+  - name: CoreDNS
+    exporters:
+      - rules:
+          - name: CoreDNS Panic Count
+            description: Number of CoreDNS panics encountered
+            query: 'coredns_panic_count_total > 0'
```

Comment:
We should alert when metrics increases here.

Can you add the exporter name such as "Embedded CoreDNS exporter" + `doc_url: https://github.com/coredns/coredns/tree/master/plugin/metrics` plz ?